src/js/types/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const content: any;
```

Comment:
should content be type `string` since that's what all the svgs should be?

Reply:
Could be a string, could be an image, maybe others? I'm just using the suggested definition webpack's docs provided.